cypress/component/ERACommons/era_commons.spec.tsx

```diff
@@ -1,15 +1,22 @@
-import ERACommons from 'src/components/era_commons/ERACommons.jsx'
-import { decodeNihToken } from 'src/utils/ERACommonsUtils.js'
-import { AuthenticateNIH } from 'src/libs/ajax/AuthenticateNIH.js'
-import { User } from 'src/libs/ajax/User.js'
-import { Storage } from 'src/libs/storage.js'
+import ERACommons from 'src/components/era_commons/ERACommons'
+import { decodeNihToken } from 'src/utils/ERACommonsUtils'
+import { AuthenticateNIH } from 'src/libs/ajax/AuthenticateNIH'
+import { User } from 'src/libs/ajax/User'
+import { Storage } from 'src/libs/storage'
 import { mount } from 'cypress/react'
 import React from 'react'
 import { Buffer } from 'buffer'
 
-// Example token from https://broad-shibboleth-prod.appspot.com/dev/login?return-url=%2Fexample-return%3Ftoken%3D%3Ctoken%3E
-const encodedToken
-  = `eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ
+interface Researcher {
+  id: number
+  firstName: string
+  lastName: string
+  email: string
+  eraCommonsId?: string
+  properties?: Array<{ propertyKey: string, propertyValue: string | number }>
+}
+
+const encodedToken = `eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ
```

Comment:
Comment on provenance seems worth retaining:

```suggestion
// Example token from https://broad-shibboleth-prod.appspot.com/dev/login?return-url=%2Fexample-return%3Ftoken%3D%3Ctoken%3E
const encodedToken = `eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ
```

Reply:
This will be removed in an upcoming RAS cleanup PR so I think this is not a pressing concern.